src/common/crypto/mbedtls/cryptoprovider.cpp

```diff
@@ -137,29 +137,25 @@ static Error ParsePrivateKey(const String& pemCAKey, mbedtls_pk_context& privKey
     mbedtls_entropy_context  entropy;
 
     mbedtls_ctr_drbg_init(&ctrDrbg);
+    auto freeDRBG = DeferRelease(&ctrDrbg, mbedtls_ctr_drbg_free);
```

Comment:
rename `crtDrbg` to `crtDrbgCtx` for example and then use: `auto crtDrbg = `

Reply:
I didn't understand the comment.